### PR TITLE
Add support for text files as subjects

### PR DESCRIPTION
**Pull request by eatyourgreens**
Describe your changes.
Adds a really simple viewer to load in a text file and render the contents.
To test, there's a file with text content as the second frame for this Diagnosis London subject.
https://text-subjects.pfe-preview.zooniverse.org/projects/eatyourgreens/diagnosis-london/talk/100/47640?env=production

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch?
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
A really simple viewer to load and render the contents of a text file.

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Just need to fix CORS errors that stop the subject file from loading.

**Comment by eatyourgreens** [comment]
Assigning a bunch of people, but whoever has time to review this really. Feel free to unassign if you're busy.

File: app/components/text-viewer.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      </div>
+    );
+  }
+}
+
+TextViewer.propTypes = {
+  src: React.PropTypes.string.isRequired,
+  type: React.PropTypes.string.isRequired,
+  format: React.PropTypes.string.isRequired,
+  onLoad: React.PropTypes.func.isRequired,
```
Comment (eatyourgreens) [inline app/components/text-viewer.jsx]: This should probably get called if the `fetch()` call is successful.

File: app/components/text-viewer.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        {this.state.content}
+      </div>
+    );
+  }
+}
+
+TextViewer.propTypes = {
+  src: React.PropTypes.string.isRequired,
+  type: React.PropTypes.string.isRequired,
+  format: React.PropTypes.string.isRequired,
```
Comment (rogerhutchings) [inline app/components/text-viewer.jsx]: The type and format props aren't being used

**Pull request merged**